project-base/app/src/FrontendApi/Resolver/Image/Exception/ImageTypeInvalidUserError.php

```diff
@@ -1,21 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace App\FrontendApi\Resolver\Image\Exception;
-
-use Shopsys\FrontendApiBundle\Model\Error\EntityNotFoundUserError;
-use Shopsys\FrontendApiBundle\Model\Error\UserErrorWithCodeInterface;
-
-class ImageTypeInvalidUserError extends EntityNotFoundUserError implements UserErrorWithCodeInterface
```

Comment:
it is out of the scope, however, it seems like `Shopsys\FrontendApiBundle\Model\Resolver\Image\Exception\ImageTypeInvalidUserError` is not used anywhere, could you please check it and remove it as well if possible? (also the "image-type-invalid" string occurrences)

Reply:
removed a503582752